Makefile

```diff
@@ -15,7 +15,7 @@ FILES= \
   semgrep_output_$(VER).py \
   semgrep_output_$(VER).ts \
   semgrep_output_$(VER).jsonschema \
-  semgrep_output_$(VER).proto \
+  protos/semgrep_output_$(VER).proto \
```

Comment:
why do we need its own folder? Can you add a README or notes.txt in the protos/ folder why we need that?

Reply:
Whoops, thanks for the reminder. I think originally moved this into `protos/` so that it'd play nice with `semgrep-app` (it expects protos to live inside a protos folder), but I'm not sure if this is necessary anymore. Let me refresh my memory and tweak or comment accordingly. (cc @ajbt200128)

Reply:
App worked w/out the extra protos dir I believe, so we should be fine on that end